test/v3/addLiquidityBoosted/addLiquidityBoosted.integration.test.ts

```diff
@@ -53,7 +55,11 @@ describe('Boosted AddLiquidity', () => {
     const addLiquidityBoosted = new AddLiquidityBoostedV3();
 
     beforeAll(async () => {
-        ({ rpcUrl } = await startFork(ANVIL_NETWORKS[ChainId[chainId]]));
+        ({ rpcUrl } = await startFork(
+            ANVIL_NETWORKS[ChainId[chainId]],
+            undefined,
+            7562540n, // block after new composite liquidity router deployed
```

Comment:
Did you try bumping the global blockNumber instead? Tests are more performant if we have all running on the same block.
If we still have other tests that are block dependent, then maybe hardcode those with the previous block and add an issue to try to improve them later.